packages/contracts-bedrock/scripts/upgrades/holocene/templates/sc-ops-task/README.md

```diff
@@ -0,0 +1,33 @@
+# Holocene Hardfork Upgrade
+
+Status: DRAFT
```

Comment:
superchain-ops is opinionated on the exact phrasing here: https://github.com/ethereum-optimism/superchain-ops/blob/516972da624b5f3137b239baabf68caa117ae440/script/utils/check-task-statuses.sh#L4
```suggestion
Status: DRAFT, NOT READY TO SIGN
```